lib/ui/onboard/onboarding_page.dart

```diff
@@ -273,6 +273,7 @@ class OnboardingText extends StatelessWidget {
                     style: EnvoyTypography.body.copyWith(
                         height: 1.2,
                         fontSize: 20,
+                        fontWeight: FontWeight.w500,
```

Comment:
To get the exact weight unfortunately we can't use `copyWith` due to https://github.com/material-foundation/flutter-packages/issues/35

Please use https://github.com/Foundation-Devices/envoy/blob/35a1d6b9bbc320dedb50b9702380f8b1f402d67e/lib/ui/theme/envoy_typography.dart#L13 instead

Reply:
It appears that the OnboardingText widget doesn't have access to material theming. I tried using setWeight from the `WeightBugWorkaround` extension, but it doesn't work because the widget can't access material theming. We can fix it by wrapping the widget in a Material widget.